spec/jenkins_pipeline/app_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe JenkinsPipeline::App do
+
+  it "Returns urls from all pipeline config files" do
+    get '/api/pipelines/ids'
+    # expect(subject).to have_received(:configuration)
```

Comment:
The idea was to test the controller itself, but I discovered Sinatra does not accept a mock at this level of testing and makes the test kind of unuseful. 
Right now it just asserting we have an endpoint working, we could use as a smoke test if we need.
So keep it or remove it, what do you think?